### PR TITLE
New Docs

### PR DESCRIPTION
**Pull request by enoldev**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,67 +2,30 @@
 description: StreamingFast Substreams documentation
```
Comment (mdimercurio) [inline docs/README.md]: we might not need a documentation here as the title already mentions "Substreams", it seems a bit repetitive
image

File: docs/README.md [hunk 2]
```diff
@@ -2,67 +2,30 @@
...
 ---
 
-# Substreams
+Substreams is a technology that allows you to **extract blockchain data in a fast and reliable way!**
 
-## Welcome to Substreams documentation
+<figure><img src=".gitbook/assets/intro/supported-chains.png" width="100%" /></figure>
 
-Substreams is a powerful blockchain indexing technology, developed for [The Graph Network](https://thegraph.com).
+**In order to define which specific data you want to extract from the blockchain, you can use Substreams packages**. For example, consider that you want to retrieve data from the Uniswap v3 smart contract. You can simply use the [Uniswap v3 Substreams Package](https://substreams.dev/streamingfast/uniswap-v3/v0.2.7) and send that data wherever you want!
```
Comment (mdimercurio) [inline docs/README.md]:
```suggestion
**Uou can use Substreams packages to define which specific data you want to extract from the blockchain**. For example, consider that you want to retrieve data from the Uniswap v3 smart contract. You can simply use the [Uniswap v3 Substreams Package](https://substreams.dev/streamingfast/uniswap-v3/v0.2.7) and send that data wherever you want!
```

File: docs/README.md [hunk 3]
```diff
@@ -2,67 +2,30 @@
...
 
-> Substreams enables developers to write Rust modules, composing data streams alongside the community, and provides extremely high-performance indexing by virtue of parallelization, in a streaming-first fashion.
->
-> Substreams have all the benefits of StreamingFast Firehose, like low-cost caching and archiving of blockchain data, high throughput processing, and cursor-based reorgs handling.
+## Consume Substreams
 
-### Where to start
+There are many ready-to-use Substreams packages, so you can simply consume them. Use the **Substreams Registry ([https://substreams.dev](https://substreams.dev)) to explore packages**.
```
Comment (mdimercurio) [inline docs/README.md]:
```suggestion
There are many ready-to-use Substreams packages, so you can simply consume them. Use the **[Substreams.dev Registry](https://substreams.dev) to explore packages**.
```

File: docs/README.md [hunk 4]
```diff
@@ -2,67 +2,30 @@
...
-Substreams uses the [Apache 2.0](../LICENSE/) license.
-
-### Disclaimer
-
-The content in the Substreams documentation was created through StreamingFast's full effort. It is up to the reader to validate the accuracy of all content presented. Substreams is in active development and, at times, the associated documentation becomes outdated. [Contact StreamingFast](https://discord.gg/mYPcRAzeVN) to report problems or service interruptions.
-
-{% hint style="info" %}
-**Note**: The Substreams documentation uses the [Google developer documentation style guide](https://developers.google.com/style) for its style and formatting.
-{% endhint %}
+{% embed url="https://www.youtube.com/watch?v=gVqGCqKVM08" %}
```
Comment (mdimercurio) [inline docs/README.md:40]: there's something weird with the video. When listening with my headset, the sound only comes from the left ear

File: docs/README.md [hunk 5]
```diff
@@ -2,67 +2,30 @@
...
+Get an overview of Substreams
+{% endembed %}
```
Comment (mdimercurio) [inline docs/README.md:42]: I'd add a text version at the bottom for folks that don't want to check the video.  Adding the sample code

File: docs/README.md [hunk 6]
```diff
@@ -2,67 +2,30 @@
 description: StreamingFast Substreams documentation
 ---
 
-# Substreams
+Substreams is a technology that allows you to **extract blockchain data in a fast and reliable way!**
```
Comment (mdimercurio) [inline docs/README.md]: I like the way Substreams is positioned in the glossary, we should include some of that content here

File: docs/new/develop/architecture.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
+description: Lenar about the Substreams architecture
```
Comment (mdimercurio) [inline docs/new/develop/architecture.md]:
```suggestion
description: Learn about the Substreams architecture
```

File: docs/new/develop/architecture.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (mdimercurio) [inline docs/new/develop/architecture.md:1]: on the site, this renders as the home page: https://substreams.streamingfast.io/~/revisions/7wlUfBToYnQ1WnIvo7uu/documentation/introduction-1/architecture

File: docs/new/develop/architecture.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+The two steps involved during parallel execution are **backward execution and forward execution**.
+
+Backward parallel execution consists of executing in parallel block ranges, from the module's initial block, up to the start block of the request. If the start block of the request matches the module's initial block no backward execution is performed.
+
+Forward parallel execution consists of executing in parallel block ranges from the start block of the request up to the last known final block, also called an irreversible block, or the stop block of the request depending on which is smaller. Forward parallel execution significantly improves the performance of Substreams.
+
+Backward parallel execution will occur in both development and production modes.
+
+Forward parallel execution only occurs in production mode.
```
Comment (mdimercurio) [inline docs/new/develop/architecture.md:37]: this doc seems very detailed for an "Architecture" doc. I'd expect a diagram of the components and definition of their role

File: docs/SUMMARY.md [hunk 10]
```diff
@@ -7,74 +7,45 @@
...
 
 * [Chains and endpoints](reference-and-specs/chains-and-endpoints.md)
 * [Substreams CLI reference](reference-and-specs/command-line-interface.md)
-* [Authentication](reference-and-specs/authentication.md)
-* [Manifests](reference-and-specs/manifests.md)
-* [Packages](reference-and-specs/packages.md)
-* [Rust crates](reference-and-specs/rust-crates.md)
+* [Manifests Reference](reference-and-specs/manifests.md)
+* [GUI Reference](reference-and-specs/manifests.md)
 * [Examples](reference-and-specs/examples.md)
```
Comment (mdimercurio) [inline docs/SUMMARY.md]: I'd try to make these examples more visible, maybe adding a link on the home page

**Review by mdimercurio — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 5 above)
Comment (enoldev) [reply docs/README.md:42]: I've included the text version in the "Develop Substreams". I'd like to keep the landing page as short as possible, so we don't stress new users

**Pull request merged**